tests/integration/test_crunch_cube.py

```diff
@@ -1932,3 +1932,9 @@ def test_values_services(self):
             [0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
         ])
         np.testing.assert_almost_equal(actual, expected)
+
+    def test_mr_x_mr_range_labels(self):
```

Comment:
@percious Please check if additional tests are needed (you're welcome to add them).

Reply:
yes, Range should have its own unit tests.